source/minimessage/format.rst

```diff
@@ -270,3 +270,21 @@ Examples
 
 .. image:: /minimessage/images/font_1.png
    :alt: The result of parsing ``Nothing <font:uniform>Uniform <font:alt>Alt  </font> Uniform``, shown in-game in the Minecraft client's chat window
+
+New Line
+********
+
+Allows you to insert a new line
+
+Tag
+   :mm:`<br>`
```

Comment:
I'd swap this around, make `<newline>` be the default with `<br>` being the alias.